### DISCUSSION TITLE
What is the issue

### DISCUSSION THREAD

**Discussion by Tommmo6** [TomSchimansky/CustomTkinter#715] [Q&A]
Exception has occurred: RuntimeError
Too early to use font: no default root window.

When I updated to 5.0.
When I create my first CTkButton this exception occurs, it was not an issue before is there something I should be doing to stop this

The button
CT.CTkButton(master=Mainwindow , state = DISABLED, corner_radius= 20 , fg_color = vars(Colour)[key] , hover_color= vars(Colour)[key] ,border_width = 2 , bg_color = "white" ,  text = ""  )

**Comment by TomSchimansky** [comment]
Can you post a complete example program?
